src/blueberrypy/config.py

```diff
@@ -115,23 +133,24 @@ def __init__(self, config_dir=None, app_config=None, logging_config=None,
         if app_config:
             self._app_config = dict(app_config)
 
-        # Merge JSON from environment variable
-        self._app_config = self.__class__.merge_dicts(self._app_config, ENV_CONFIG)
-
-        # Convert relative paths to absolute where needed
-        try:
-            for _ in self._app_config['controllers']:
-                section = self._app_config['controllers'][_]
-                for r in section:
-                    if isinstance(section[r], dict):
-                        for __ in ['tools.staticdir.root',
-                                   'tools.staticfile.root']:
-                            pth = section[r].get(__)
-                            if pth is not None and not pth.startswith('/'):
-                                self._app_config['controllers'][_][r][__] = \
-                                    os.path.join(CWD, pth)
-        except:
-            pass
+        if hasattr(self, '_app_config'):
```

Comment:
This is useless. There **has to be** an app config at this point either from `app_config` argument or from config files.
Remove it.